src/matrix/wishart.jl

```diff
@@ -229,13 +235,14 @@ function _univariate(d::Wishart)
     check_univariate(d)
     df, S = params(d)
     α = df / 2
-    β = 2Matrix(S)[1]
+    β = 2 * first(S)
     return Gamma(α, β)
 end
 
 function _rand_params(::Type{Wishart}, elty, n::Int, p::Int)
     n == p || throw(ArgumentError("dims must be equal for Wishart"))
-    ν = elty( n - 1 + abs(10randn()) )
-    S = (X = 2rand(elty, n, n) .- 1; X * X')
+    ν = elty(n - 1 + abs(10 * randn()))
+    X = 2 .* rand(elty, n, n) .- 1
```

Comment:
Reduced allocations:
```suggestion
    X = rand(elty, n, n)
    X .= 2 .* X .- 1
```